docs/content/guides/migrations/migrating-airflow-to-dagster.mdx

```diff
@@ -5,223 +5,242 @@ description: "Learn how do a lift-and-shift migration of airflow to Dagster."
 
 # Migrating Airflow to Dagster
```

Comment:
How do we feel about the structure of this page - do we need an index at the top? Should it be split into multiple pages?

We will continue to append new migration strategies / mapping of concepts over time.